config/manager/manager.yaml

```diff
@@ -48,3 +48,9 @@ spec:
           initialDelaySeconds: 5
           periodSeconds: 10
       terminationGracePeriodSeconds: 10
+      serviceAccountName: nexus-operator-sa
```

Comment:
*this* one _does_ need to be `nexus-operator-sa` as `kustomize` isn't smart enough to recognize that this is a pointer to an attribute on a `ServiceAccount` object